### PR TITLE
Gina Christians DOM -II

### PR DESCRIPTION
**Pull request by Chrisgins**
The assignment is complete but the code only partially works. What works in codepen doesn't work in the browser and some doesn't work at all. I will try to rework the code but for now, this is what I have.

### CODE REVIEW

**Comment by Chrisgins** [comment]
reworked MVP portion of the assignment and got everything to work; went on to stretch but couldn't get it to work as expected.

File: js/index.js [hunk 1]
```diff
@@ -1 +1,126 @@
-// Your code goes here
+
+//==== Event 1: change to halloween picture on dblclick to document====
+
+
+document.addEventListener('dblclick', () => {
```
Comment (trevorfehrman) [inline js/index.js]: Nice

Comment (trevorfehrman) [reply js/index.js]: Clever use of window.open

File: js/index.js [hunk 2]
```diff
@@ -1 +1,126 @@
...
+  // console.log('I put a spell on you, and now you're mine!)
+  window.open('https://cdn.pixabay.com/photo/2018/10/16/10/15/halloween-3751095_1280.jpg');
+})  
+
+
+//====Event 2: Randomize background color on nav ====
+
+function getRandomColor() {
```
Comment (trevorfehrman) [inline js/index.js]: Very cool

File: js/index.js [hunk 3]
```diff
@@ -1 +1,126 @@
...
+    const randomColor = Math.floor(Math.random() * 256);
+    colors.push(randomColor);
+  }
+  return `rgb(${colors[0]}, ${colors[1]}, ${colors[2]})`;
+}
+
+ let navBar = document.querySelector('.nav');
+  navBar.addEventListener('click', (event) => {
+  event.target.style.backgroundColor = getRandomColor();
+  event.stopPropagation();
```
Comment (trevorfehrman) [inline js/index.js]: Is stop propagation needed here?

File: js/index.js [hunk 4]
```diff
@@ -1 +1,126 @@
...
+});  
+
+
+//====Event 3: Change sign-up btns to read 'Hi Ash!'====
+
+let btnText = document.querySelectorAll('.content-pick .btn');
+btnText.forEach(btn => {
+  btn.addEventListener('click', e => {
+    e.stopPropagation();
+    btn.innerText = 'Hi Ash!';
```
Comment (trevorfehrman) [inline js/index.js]: I'll pass this along for you

File: js/index.js [hunk 5]
```diff
@@ -1 +1,126 @@
...
+    false
+    ); 
+
+
+ //====Event 7: Add no copying event listener====
+
+ document.addEventListener('copy',nope);
+
+function nope(){
+    alert("Swiper no swiping!!");
```
Comment (trevorfehrman) [inline js/index.js]: hahaha

File: js/index.js [hunk 6]
```diff
@@ -1 +1,126 @@
...
+
+
+//====Event 10: Change nav to green text and increase size on dbl click====
+
+let header1 = document.querySelector('.main-navigation');
+
+
+console.log(header1);
+header1.addEventListener('dblclick', function(event) {
+    event.stopPropagation();
```
Comment (trevorfehrman) [inline js/index.js]: I'm not sure if lines 120 and 121 are needed

File: stretch-assignment/index.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
+
+
+//====Add container event listener====
+
+const container = document.querySelector('.blocks');
+container.addEventListener('click', e => {
+  let firstBlock = e.target;
+   container.prepend(firstBlock);
```
Comment (trevorfehrman) [inline stretch-assignment/index.js]: very cool

**Pull request closed without merging**